packages/calcite-components/src/components/sheet/sheet.scss

```diff
@@ -3,7 +3,11 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-sheet-scrim-background: Specifies the background color of the sheet scrim.
+ * @prop --calcite-sheet-scrim-background: [Deprecated] use `--calcite-sheet-scrim-background-color` instead. Specifies the background color of the sheet scrim.
+ * @prop --calcite-sheet-scrim-background-color: Specifies the background color of the sheet scrim.
+ * @prop --calcite-sheet-background-color: Specifies the background color of the sheet.
```

Comment:
I wonder if this needs to have a BG set at all. I'd probably vote to just remove this and have the container display with transparent BG. A user should be slotting a Panel, etc., so this doesn't seem needed.

Reply:
This would apply to modal too.

Reply:
Well, Modal does currently have bg color (and separate for Content area). It’s a bit different than Sheet as you don’t slot a Panel into a Modal. That is the current behavior anyway, we have discussed changing that to align more with Sheet, Shell panel, etc.

Reply:
Gotcha. Do we have an issue for that? For future reference purposes.

Reply:
would that apply to shell panel too?

Reply:
#7886 would address that I think.